scenario/go.mod

```diff
@@ -0,0 +1,79 @@
+module github.com/kubernetes-sigs/kube-scheduler-simulator/scenario
```

Comment:
This PR creates it as another module. You may want to discuss it.

My point is:
- by creating it as a separate module, we can completely separate the logic. It may help us in the future like when we want to expand it to be run in user's cluster. 
  - And of course, it makes it easier to maintain. (those who want to develop this controller don't need to know about simulator's implementation.)
- we can run the controller in another container of the simulator's Pod. 
  - docker-compose as well.

Reply:
> we can run the controller in another container of the simulator's Pod

It requires this change on KEP: https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/194

Reply:
Yeah, upon #194 getting merged, I agree that we make the controller the new module.
I ack-ed https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/196/commits/24a77a823391896ed1a8c340844e6f1b70b5c9ec changed the module name.